managed/Makefile

```diff
@@ -33,11 +33,11 @@ clean:                          ## Remove generated files.
 	find . -name mock_*.go -delete
 
 release:                        ## Build pmm-managed release binaries.
-	env CGO_ENABLED=0 go build -v $(PMM_LD_FLAGS) -o $(PMM_RELEASE_PATH)/ ./...
+	env CGO_ENABLED=0 go build -v $(PMM_LD_FLAGS) -o $(PMM_RELEASE_PATH)/ ./cmd/...
```

Comment:
fixes issue in linux:
--- FAIL: TestStarlarkSandbox (60.02s)
    main_test.go:107: 
        	Error Trace:	/root/go/src/github.com/percona/pmm/managed/cmd/pmm-managed-starlark/main_test.go:107
        	Error:      	Received unexpected error:
        	            	exit status 2
        	Test:       	TestStarlarkSandbox
FAIL
FAIL	github.com/percona/pmm/managed/cmd/pmm-managed-starlark	60.095s

Reply:
plus, it should be faster